Makefile

```diff
@@ -155,21 +155,35 @@ dbuild: crioimage
 integration: crioimage
 	$(CONTAINER_RUNTIME) run -e STORAGE_OPTIONS="--storage-driver=vfs" -e TEST_USERNS -e TESTFLAGS -e TRAVIS -t --privileged --rm -v ${CURDIR}:/go/src/${PROJECT} ${CRIO_IMAGE} make localintegration
 
-${BUILD_BIN_PATH}/ginkgo:
+GO_MD2MAN := ${BUILD_BIN_PATH}/go-md2man
+GINKGO := ${BUILD_BIN_PATH}/ginkgo
+MOCKGEN := ${BUILD_BIN_PATH}/mockgen
+
+define go-build
+	$(GO) build -o ${BUILD_BIN_PATH}/${1} ${2}
+endef
+
+${BUILD_BIN_PATH}:
 	mkdir -p ${BUILD_BIN_PATH}
-	$(GO) build -o ${BUILD_BIN_PATH}/ginkgo ./vendor/github.com/onsi/ginkgo/ginkgo
+
+${GO_MD2MAN}: ${BUILD_BIN_PATH}
+	$(call go-build,go-md2man,./vendor/github.com/cpuguy83/go-md2man)
+
+${GINKGO}: ${BUILD_BIN_PATH}
+	$(call go-build,ginkgo,./vendor/github.com/onsi/ginkgo/ginkgo)
+
+${MOCKGEN}:
+	$(call go-build,mockgen,./vendor/github.com/golang/mock/mockgen)
 
 vendor: .install.vndr
 	$(GOPATH)/bin/vndr \
-		-whitelist "github.com/onsi/ginkgo" \
-		-whitelist "github.com/golang/mock" \
+		-whitelist github.com/onsi/ginkgo \
+		-whitelist github.com/golang/mock \
+		-whitelist github.com/cpuguy83/go-md2man \
+		-whitelist github.com/russross/blackfriday \
```

Comment:
are you using blackfriday with this PR?  Or is this something you need in place for later on?

Reply:
Yes we need it as direct dependency of `go-md2man` and have to whitelist it there to avoid removal during running `make vendor`.

Reply:
OK, maybe we're calling/using go-md2man in Podman and Buildah differently, but we're not pulling in blackfriday in either of those projects that I can see.